Customer/add sub item

fivetran-joemarkiewicz opened the pull request:
**Are you a current Fivetran customer?** 
<!--- Please tell us your name, title and company -->
Fivetran created PR

**What change(s) does this PR introduce?** 
<!--- Describe what changes your PR introduces to the package and how to leverage this new feature. -->

## Under the Hood
- The `plan_interval` field within the `stg_stripe__plan` model has been aliased within the `get_plan_columns()` macro to allow for more flexibility across warehouses.
- BuildKite testing has been added.

**Did you update the CHANGELOG?** 
<!--- Please update the new package version’s CHANGELOG entry detailing the changes included in this PR. -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [X] Yes

**Does this PR introduce a breaking change?**
<!--- Does this PR introduce changes that will cause current package users' jobs to fail or require a `--full-refresh`? -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide an explanation as to how the change is non-breaking below.)

This is strictly an under the hood update and will not cause a breaking change for existing users.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 
<!--- The dbt_project.yml and the integration_tests/dbt_project.yml files contain the version number. Be sure to upgrade it accordingly -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**
<!--- If an Issue was created it is helpful to track the progress by linking it in the PR. -->
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [X] Yes, this PR is needed for the release of [dbt_stripe PR#52](https://github.com/fivetran/dbt_stripe/pull/52)
- [ ] No 

**How did you test the PR changes?** 
<!--- Proof of testing is required in order for the PR to be approved. -->
<!--- To check a box, remove the space and insert an x in the box (eg. [x] CircleCi). --> 
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [X] BuildKite <!--- BuildKite testing is only applicable to Fivetran employees. --> 
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
<!--- To check a warehouse remove the space and insert an x in the box (eg. [x] Bigquery). --> 
<!--- To select a checkbox you simply need to add an "x" with no spaces between the brackets (eg. [x] Yes). -->
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [X] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
<!--- For a complete list of markdown compatible emojis check our this git repo (https://gist.github.com/rxaviers/7360908)  --> 
🚙 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

fivetran-sheringuyen reviewed (commented):
Hey @fivetran-joemarkiewicz, thanks for working on this PR! I've made a couple in-line notes and had one extra thing -- I think this is a change that will warrant a docs regeneration.

fivetran-joemarkiewicz commented on .buildkite/hooks/pre-command:
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo
```
Yup this is a holdover from me trying to figure out the pipeline fail before we talked during standup. Corrected.

fivetran-joemarkiewicz commented on CHANGELOG.md:
```diff
@@ -1,3 +1,7 @@
+# dbt_stripe_source v0.7.4
+## Under the Hood
+- The `plan_interval` field within the `stg_stripe__plan` model has been aliased within the `get_plan_columns()` macro to allow for more flexibility across warehouses. ([#58](https://github.com/fivetran/dbt_stripe_source/pull/58))
+- BuildKite testing has been added. ([#58](https://github.com/fivetran/dbt_stripe_source/pull/58))
```
This being the BuildKite testing? I don't think it is to necessary knowing that it will be an additional item. I just added it here since there were a number of files added outside of the migration updates and wanted to call attention to that.

fivetran-joemarkiewicz commented on integration_tests/ci/sample.profiles.yml:
```diff
@@ -1,13 +1,13 @@
 
-# HEY! This file is used in the stripe integrations tests with CircleCI.
+# HEY! This file is used in the dbt package integrations tests with CircleCI.
```
Updated

fivetran-joemarkiewicz commented on integration_tests/requirements.txt:
```diff
@@ -1,6 +1,7 @@
-dbt-spark[PyHive]~=1.0.0
+dbt-snowflake>=1.0.0,<2.0.0
+dbt-bigquery>=1.0.0,<2.0.0
+dbt-redshift>=1.0.0,<2.0.0
```
I don't think it is too much of a risk with us having the upper bound being 2.0.0 since this is only for integration testing and will not be used once we push the migration updates. However, for consistency I feel it makes sense to adjust. Updated.

fivetran-joemarkiewicz commented:
> Hey @fivetran-joemarkiewicz, thanks for working on this PR! I've made a couple in-line notes and had one extra thing -- I think this is a change that will warrant a docs regeneration.

Thanks @fivetran-sheringuyen just addressed the review notes! I also did wonder if this would require a docs regen; however, the only user side change is where we are doing the renaming of `interval` to `plan_interval` the only difference is the renaming is happening in the `fields` cte as opposed to the `final` cte of the [stg_stripe__plan](https://github.com/fivetran/dbt_stripe_source/blob/main/models/stg_stripe__plan.sql) model. Therefore, the docs wouldn't be affected. 

However, now that I say that.... the code still changed 😮‍💨 I'll regen the docs.

fivetran-sheringuyen commented:
Sweet! LGTM, as long as Buildkite passes we should be good to go
